templates/federated_reporting/import.sh

```diff
@@ -100,6 +102,22 @@ else
   done
 fi
 
+if [ "$CFE_FR_HANDLE_DUPLICATES" = "yes" ]; then
+  log "Handle Duplicate Hostkeys"
+  hostkey_list=$(printf "'%s'," ${hostkeys[*]} | sed -e 's/,$//')
+  "$CFE_BIN_DIR"/psql -U $CFE_FR_DB_USER -d cfdb --set "ON_ERROR_STOP=1" \
+                      -c "SELECT handle_duplicate_hostkeys_in_import(ARRAY[$hostkey_list]);"  \
+    > duplicates.log || failed=1
+  if [ "$failed" = "0" ]; then
+    log "Handle Duplicate Hostkeys: DONE"
+  else
+    log "Handle Duplicate Hostkeys: FAILED"
+    log "last 10 lines of duplicates.log"
+    tail -n 10 duplicates.log
+    exit 1
```

Comment:
I don't think this should prevent the further steps in the import process below.

Reply:
I am inclined to think it should fail. If we don't fail it then the database could be in an odd state, especially if the users are expecting "no duplicates" and then suddenly there are a lot of duplicates.

If we `exit 1` then that will show up and could be handled like a "warning" to a sysadmin right?

Reply:
Yes, but it will also mean they will have no data from the problematic feeder(s) at all. But then again, this only happens if they explicitly enable de-duplication, right? In that case it's OK to `exit` here, I think. :+1: